packages/components/src/card/card/index.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Do we really want to put this in its own directory? `/card/card` seems a bit odd 😉  Should we move this component a directory up?

Reply:
This folder structure was agreed with @sarayourfriend. The same approach was used for the `<Flex />` component which sits in the [`/packages/components/src/flex/flex` folder](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/flex/flex)

Reply:
Consistency FTW 👍  Let's go with that then!